<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rust::Internal::BuildStepConfigWidget</class>
 <widget class="QWidget" name="Rust::Internal::BuildStepConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>67</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="Utils::FancyLineEdit" name="extraArgs"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="extraArgsLabel">
       <property name="text">
        <string>Extra arguments:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="showJsonOutput">
     <property name="text">
      <string>Show JSON output on console</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/fancylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
